archlinuxcn/sendanywhere/lilac.yaml

```
@@ -0,0 +1,13 @@
+maintainers:
+  - github: Xeonacid
+    email: h.dwwwwww@gmail.com
+
+build_prefix: extra-x86_64
+
+pre_build: aur_pre_build
+
+post_build: aur_post_build
+
+update_on:
+  - source: aur
+    aur: sendanywhere
```

@lilydjwg is the following hunk okay?
```suggestion
update_on:
  - source: httpheader
    url: https://update.send-anywhere.com/linux_downloads/sendanywhere_latest_amd64.deb
    header: last-modified
    regex: ".*"
```

---

Yes.